.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
```

Comment:
Wonder if wouldn't need to create another entry for dependabot to track requirements files inside the `requirements` directory.

Reply:
Based on my experiment it found the subdirectories automatically.